diff --git a/deps/lcb/src/utilities.h b/deps/lcb/src/utilities.h
--- a/deps/lcb/src/utilities.h
+++ b/deps/lcb/src/utilities.h
@@ -21,6 +21,7 @@
 #include "config.h"
 
 #ifdef __cplusplus
+#include <cstdint>
 #include <string>
 #include <vector>
 
diff --git a/deps/lcb/include/libcouchbase/couchbase.h b/deps/lcb/include/libcouchbase/couchbase.h
--- a/deps/lcb/include/libcouchbase/couchbase.h
+++ b/deps/lcb/include/libcouchbase/couchbase.h
@@ -36,6 +36,9 @@
 typedef struct lcb_HTTP_HANDLE_ lcb_HTTP_HANDLE;
 
 #include <stddef.h>
+#ifdef __cplusplus
+#include <cstdint>
+#endif
 #include <time.h>
 #include <stdarg.h>
 #include <stdio.h>
